meteor/lib/collections/lib.ts

```diff
@@ -81,17 +76,19 @@ export function ObserveChangesForHash<DBInterface extends { _id: ProtectedString
 	}
 }
 
+/**
+ * Create a fully featured MongoCollection
+ * @param name Name of the collection in mongodb
+ * @param options Open options
+ */
```

Comment:
No issues here, just a thanks and a 👍  for taking the time to improve the docs as you go! :)